assets/js/modules/adsense/components/dashboard/AdSenseDashboardWidget.js

```diff
@@ -193,7 +196,7 @@ export default function AdSenseDashboardWidget() {
 						</Cell>
 
 						<Cell alignRight size={ 12 }>
-							<HelpLink />
+							{ ! helpVisibilityEnabled && <HelpLink /> }
 						</Cell>
```

Comment:
Same here - also check other components throughout this PR, I won't annotate the other ones.

This problem also is present in components that you already changed in the previous PR, **please update the `PageFooter` component as well**.